understanding/index.html

```diff
@@ -113,21 +113,20 @@
             				
             		
             
-            <h1 id="title" class="title p-name">All WCAG 2.2 Understanding Docs</h1>
+            <h1 id="title" class="title p-name">WCAG 2.2 解説書目次</h1>
             				
             
             <aside class="box">
                					
                
-               <header class="box-h ">Summary</header>
+               <header class="box-h ">要約</header>
                					
                
                <div class="box-i">
                   						
                   
-                  <p><em>Understanding documents</em> provide detailed explanations for Web Content Accessibility Guidelines (WCAG) guidelines
-                     and success criteria. They are informative, not part of the "normative" WCAG standard.
-                     For information, see <a href="about">About WCAG Understanding Documents</a>.
+                  <p><em>解説書</em>は、ウェブコンテンツアクセシビリティガイドライン (WCAG) のガイドライン及び達成基準に関する詳細な説明を提供する。これらは参考情報であり、「規範的な」WCAG 標準の一部ではない。
```

Comment:
Web Content Accessibility Guidelinesは英語のままにしていたように思います。

```suggestion
                  <p><em>解説書</em>は、Web Content Accessibility Guidelines (WCAG) のガイドライン及び達成基準に関する詳細な説明を提供する。これらは参考情報であり、「規範的な」WCAG 標準の一部ではない。
```